sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -357,7 +359,114 @@
+     *
+     * @param businessCardUrl The source URL to the input business card.
+     *
+     * @return A {@link SyncPoller} to poll the progress of the recognize business card operation until it has
+     * completed, has failed, or has been cancelled. The completed operation returns a list of {@link RecognizedForm}.
+     * @throws FormRecognizerException If recognize operation fails and the {@link AnalyzeOperationResult} returned with
+     * an {@link OperationStatus#FAILED}.
+     * @throws NullPointerException If {@code businessCardUrl} is null.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> beginRecognizeBusinessCardsFromUrl(
+        String businessCardUrl) {
+        return beginRecognizeBusinessCardsFromUrl(businessCardUrl, null, Context.NONE);
+    }
+
+    /**
+     * Recognizes business card data from documents using optical character recognition (OCR) and a
+     * prebuilt business card trained model.
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support</p>
+     * See <a href="https://aka.ms/formrecognizer/businesscardfields">here</a> for fields found on a business card.
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.FormRecognizerClient.beginRecognizeBusinessCardsFromUrl#string-RecognizeBusinessCardsOptions-Context}
+     *
+     * @param businessCardUrl The source URL to the input business card.
+     * @param recognizeBusinessCardsOptions The additional configurable {@link RecognizeBusinessCardsOptions options}
+     * that may be passed when analyzing a business card.
+     * @param context Additional context that is passed through the HTTP pipeline during the service call.
+     *
+     * @return A {@link SyncPoller} to poll the progress of the recognize business card operation until it has
+     * completed, has failed, or has been cancelled. The completed operation returns a list of {@link RecognizedForm}.
+     * @throws FormRecognizerException If recognize operation fails and the {@link AnalyzeOperationResult} returned with
+     * an {@link OperationStatus#FAILED}.
+     * @throws NullPointerException If {@code businessCardUrl} is null.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> beginRecognizeBusinessCardsFromUrl(
+        String businessCardUrl, RecognizeBusinessCardsOptions recognizeBusinessCardsOptions, Context context) {
+        return client.beginRecognizeBusinessCardsFromUrl(businessCardUrl, recognizeBusinessCardsOptions, context)
+                   .getSyncPoller();
+    }
+
+    /**
+     * Recognizes data from the provided document data using optical character recognition (OCR)
```

Comment:
following what you had before, consider adding ` Recognizes business card data...`

Reply:
same in other places